src/gen-ts.ts

```diff
@@ -238,15 +239,20 @@ interface TsType {
 function getTsType(type?: string): TsType {
   let optional = false;
 
-  // default to 'any'
-  type = type || 'any';
+  // default to '{}' in preference to 'any'
+  type = type || '{}';
   
   // handle Closure optionals
   if (type.endsWith('=')) {
     optional = true;
     type = type.substring(0, type.length - 1);
   }
 
+  // handle Closure unknown type
+  if (type.startsWith('?')) {
+    type = '{}';
```

Comment:
why `{}` instead of any? Does the unknown type exclude primitives?

Reply:
@justinfagnani That was just because the Google Style Guide recommends using {} instead of any. Want me to change them back?

Reply:
`any` seems more idiomatic externally.

This conditional seems like a bug though. `?X` should translate to `X|null` (in fact `X` should generate `X|null`).